CHANGES.rst

```diff
@@ -1,3 +1,11 @@
+1.11.0 (beta)
```

Comment:
ah, I'm not referring to the package as being in beta, but about 1.11.0 being in beta (and not released) - the others have release dates because they were released.

What I was trying to do was get us so that we could put more than one fix/upgrade into the next release instead of each of us having to do a release. Do we have an automated releaser or something that would make it so we should just always release a new version for each merged PR?